packages/typechain-target-ethers/README.md

```diff
@@ -0,0 +1,19 @@
+# Typechain target ethers
+
+<p align="center">
+  <img src="https://github.com/Neufund/TypeChain/blob/d82f3cc644a11e22ca8e42505c16f035e2f2555d/docs/images/typechain-logo.png?raw=true" width="300" alt="TypeChain">
+  <h3 align="center">Typechain target ethers</h3>
+  <p align="center">🔌 TypeScript bindings for ethers smartcontracts</p>
+
+  <p align="center">
+    <a href="https://circleci.com/gh/ethereum-ts/TypeChain"><img alt="Build Status" src="https://circleci.com/gh/ethereum-ts/TypeChain/tree/master.svg?style=svg"></a>
+    <a href="https://coveralls.io/github/ethereum-ts/TypeChain?branch=master"><img alt="Coverage" src="https://coveralls.io/repos/github/ethereum-ts/TypeChain/badge.svg?branch=master"></a>
+    <img alt="Downloads" src="https://img.shields.io/npm/dm/typechain.svg">
+    <a href="https://github.com/prettier/prettier"><img alt="Prettier" src="https://img.shields.io/badge/code_style-prettier-ff69b4.svg"></a>
+    <a href="/package.json"><img alt="Software License" src="https://img.shields.io/badge/license-MIT-brightgreen.svg?style=flat-square"></a>
+  </p>
+
+  <p align="center">
+    <a href="https://blog.neufund.org/introducing-typechain-typescript-bindings-for-ethereum-smart-contracts-839fc2becf22">Medium post</a> | <a href="https://www.youtube.com/watch?v=9x6AkShGkwU">DappCon Video</a>
+  </p>
+</p>
```

Comment:
We could add a link to the main TypeChain README to all target package readmes -- in case someone lands there from a google search or something.